packages/integrations/package.json

```diff
@@ -32,7 +36,7 @@
     "rollup": "^1.10.1",
     "rollup-plugin-commonjs": "^9.3.4",
     "rollup-plugin-node-resolve": "^4.2.3",
-    "rollup-plugin-terser": "^4.0.4",
+    "rollup-plugin-terser": "^7.0.0",
```

Comment:
Terser 6 dropped support for Rollup 1, but we still use it here. Please either revert plugin version, or update rollup and verify everything else still works.

Reply:
revert the version for the rollup-plugin-terser module, made this change in the new pull request.